src/turbomind/kernels/unfused_attention_kernels.cu

```diff
@@ -1521,49 +1544,27 @@ void invokeAddFusedQKVBiasTranspose(T*                               q_buf,
                                     const int                        seq_len,
                                     const int                        token_num,
                                     const int                        head_num,
+                                    const int                        kv_head_num,
                                     const int                        size_per_head,
                                     const int                        rotary_embedding_dim,
                                     const int                        neox_rotary_style,
                                     const float*                     scale,
                                     const int                        int8_mode,
                                     cudaStream_t                     stream)
 {
-    // [bs, seq_len, 3, head, Dh]
-    if (rotary_embedding_dim == 0 && param.max_prefix_prompt_length == 0) {
-        const int m = token_num;
-        const int n = head_num * size_per_head;
-        dim3      block(384);
-        dim3      grid((int)(ceil(1.0 * m * n / 384)));
-        add_fusedQKV_bias_transpose_kernel<<<grid, block, 0, stream>>>(q_buf,
-                                                                       k_buf,
-                                                                       v_buf,
-                                                                       QKV,
-                                                                       qkv_bias,
-                                                                       padding_offset,
-                                                                       batch_size,
-                                                                       seq_len,
-                                                                       token_num,
-                                                                       head_num,
-                                                                       size_per_head,
-                                                                       scale,
-                                                                       int8_mode);
+    TM_LOG_ERROR("invokeAddFusedQKVBiasTranspose");
```

Comment:
Use debug log or there will be annoy log on fmha.

Reply:
Done